lib/measured/conversion.rb

```diff
@@ -48,6 +48,9 @@ def conversion_table
 
   def unit_for(name)
     unit = @units.find { |unit| unit.names_include?(name, case_sensitive: @case_sensitive) }
-    unit or raise Measured::UnitError, "Cannot find unit for #{name}"
+  end
+
+  def unit_for!(name)
+    unit_for(name) or raise Measured::UnitError, "Cannot find unit for #{name}"
```

Comment:
The benefit of choosing a single operator is more pragmatic. It looks obvious when you write it but not necessarily when you review it. Make it clear and don't make me think, even if it needs a few more lines of code.